tests/test_engine_parser_ident_base_parser.py

```diff
@@ -281,16 +284,30 @@ def test_get_mass_and_composition():
     seq = "PEPTCIDE"
     mods = ""
     # Without modifications
-    m, comp = get_mass_and_composition(seq=seq, mods=mods)
-    assert m == pytest.approx(902.3691)
+    m_no_mods, comp = get_mass_and_composition(
+        cc=ChemicalComposition(), seq=seq, mods=mods
+    )
+    assert m_no_mods == pytest.approx(902.3691)
     assert comp == "C(37)H(58)N(8)O(16)S(1)"
 
     # With modifications
     mods = "Acetyl:0;Carbamidomethyl:5"
-    m, comp = get_mass_and_composition(seq=seq, mods=mods)
+    m, comp = get_mass_and_composition(cc=ChemicalComposition(), seq=seq, mods=mods)
     assert m == pytest.approx(902.3691 + 57.021464 + 42.010565)
```

Comment:
One could swap here the 902.3691 by `m_no_mods` to be consistent with the follow up example